src/backend/sev/mod.rs

```diff
@@ -193,7 +192,11 @@ impl backend::Backend for Backend {
         data
     }
 
-    fn build(&self, _code: Component, _sock: Option<&Path>) -> Result<Arc<dyn Keep>> {
-        unimplemented!()
+    fn build(&self, code: Component, _sock: Option<&Path>) -> Result<Arc<dyn Keep>> {
+        let shim = Component::from_bytes(SHIM)?;
+
+        let vm = Builder::new(Config::default(), shim, code, builder::Sev).build()?;
+
+        Ok(Arc::new(RwLock::new(vm)))
```

Comment:
Does this end up double-arcing the kvm?

Reply:
No. `Builder::build()` just returns a bare `struct Vm`, so this `Arc::new()` is the first and only `Arc`.